sphinxcontrib/spelling/__init__.py

```diff
@@ -19,6 +19,8 @@ def setup(app):
     app.add_directive('spelling', SpellingDirective)
     # Report guesses about correct spelling
     app.add_config_value('spelling_show_suggestions', False, 'env')
+    # Report the whole line that has the error
+    app.add_config_value('spelling_show_whole_line', False, 'env')
```

Comment:
I assume you opted to set the default to false for backwards compatibility? It seems safe enough to make it true. How would you feel about that?

Reply:
Yeah, I had it as `False` for that reason. I'm more than happy to switch to `True` and have everyone automatically get the benefit of this change!